docs/contracts/contracts/LSP0ERC725Account/LSP0ERC725Account.md

```diff
@@ -0,0 +1,1485 @@
+
+:::
+
+```solidity
+function acceptOwnership() external nonpayable;
+```
+
+_`msg.sender` is accepting ownership of contract: `address(this)`._
+
+Transfer ownership of the contract from the current [`owner()`](#owner) to the [`pendingOwner()`](#pendingowner). Once this function is called:
+
+- The current [`owner()`](#owner) will loose access to the functions restricted to the [`owner()`](#owner) only.
+
+- The [`pendingOwner()`](#pendingowner) will gain access to the functions restricted to the [`owner()`](#owner) only.
+
+<blockquote>
+
+**Requirements:**
+
+- MUST be called by the [`pendingOwner`](#pendingowner).
+- When notifying the previous owner via LSP1, the typeId used MUST be the `keccak256(...)` hash of [LSP0OwnershipTransferred_SenderNotification].
+- When notifying the new owner via LSP1, the typeId used MUST be the `keccak256(...)` hash of [LSP0OwnershipTransferred_RecipientNotification].
+
+</blockquote>
+
+### batchCalls
+
+:::note Links
+
+- Specification details in [**LSP-0-ERC725Account**](https://github.com/lukso-network/lips/tree/main/LSPs/LSP-0-ERC725Account.md#batchcalls)
+- Solidity implementation in [**LSP0ERC725Account**](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP0ERC725Account/LSP0ERC725Account.sol)
+- Function signature: `batchCalls(bytes[])`
+- Function selector: `0x6963d438`
+
+:::
+
+```solidity
+function batchCalls(bytes[] data) external nonpayable returns (bytes[] results);
+```
+
+Allows a caller to batch different function calls in one call. Perform a `delegatecall` on self, to call different functions with preserving the context It is not possible to send value along the functions call due to the use of `delegatecall`.
+
+#### Parameters
+
+| Name   |   Type    | Description                                                          |
+| ------ | :-------: | -------------------------------------------------------------------- |
+| `data` | `bytes[]` | An array of ABI encoded function calls to be called on the contract. |
+
+#### Returns
+
+| Name      |   Type    | Description                                            |
+| --------- | :-------: | ------------------------------------------------------ |
+| `results` | `bytes[]` | An array of values returned by the executed functions. |
+
+### execute
+
+:::note Links
+
+- Specification details in [**LSP-0-ERC725Account**](https://github.com/lukso-network/lips/tree/main/LSPs/LSP-0-ERC725Account.md#execute)
+- Solidity implementation in [**LSP0ERC725Account**](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP0ERC725Account/LSP0ERC725Account.sol)
+- Function signature: `execute(uint256,address,uint256,bytes)`
+- Function selector: `0x44c028fe`
+
+:::
+
+```solidity
+function execute(
+  uint256 operationType,
+  address target,
+  uint256 value,
+  bytes data
+) external payable returns (bytes);
+```
+
+_Calling address `target` using `operationType`, transferring `value` wei and data: `data`. _
```

Comment:
```suggestion
_Calling address `target` using `operationType`, transferring `value` wei and data: `data`._
```

Reply:
This is on IERC725X side because it is from `@inheritdoc IERC725X`